syz-manager/manager.go

```diff
@@ -863,9 +864,14 @@ func (mgr *Manager) saveRepro(res *manager.ReproResult) {
 
 	var cprogText []byte
 	if repro.CRepro {
-		var err error
-		cprogText, err = repro.CProgram()
-		if err != nil {
+		cprog, err := csource.Write(repro.Prog, repro.Opts)
```

Comment:
Please move this chunk into a separate PR, we could merge such small fixes/improvements much faster, and reduce diff of the main change.
The PR may include the MAINTAINERS change, and the newSignal logging change below.

Reply:
This particular change actually looks like some merge conflict with recent changes in 1a7fb460b15fc14174513a2a2217c3a5b454ac49